lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -389,6 +389,13 @@ def estimate_diffusion(self, method, max_lag=None):
         if method not in ("gls", "ols"):
             raise ValueError('Invalid method selected. Method must be "gls" or "ols"')
 
+        if not self._kymo.contiguous:
+            raise NotImplementedError(
+                "Estimating diffusion constants from data with discontiguous pixels not supported. "
+                "To estimate diffusion constants, do not downsample the kymograph temporally prior "
+                "to tracking."
+            )
```

Comment:
I wonder if we should include the little bit from the `contiguous` docstring? or somehow mention that the actual problem is integrating over disjoint time. Maybe ...
```suggestion
        if not self._kymo.contiguous:
            raise NotImplementedError(
                "Estimating diffusion constants from data which has been integrated over disjoint sections "
                "of time is not supported. To estimate diffusion constants, do not downsample the kymograph "
                " temporally prior to tracking."
            )
```

Reply:
Makes sense. It's a bit more clear. I'll go with your suggestion.